seirmo/slider.py

```diff
@@ -0,0 +1,125 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import dash_core_components as dcc
+import dash_html_components as html
+import numpy as np
+
+
+class _SliderComponent(object):
+    """Slider Class:
+
+    Controls the creation and grouping of slider for the app
+    """
+
+    def __init__(self): # noqa
+        super(_SliderComponent, self).__init__()
+
+        self._slider_ids = []
+        self._sliders = {}
+        self._group_ids = {}
+
+    def add_slider(self, slider_id, min, max, initial_value=None, step_size=None, label=None, mark_num=None): # noqa
```

Comment:
the variables `min` and `max` are protected in python, and are the built-in minimum and maximum functions of python. It's good practice to never overwrite those methods. You could just go for `min_value` or so.